iml-gui/crate/src/components/breadcrumbs.rs

```diff
@@ -0,0 +1,103 @@
+use crate::{Msg, Page};
+
+use seed::{
+    a, attrs, div,
+    dom_types::{At, El, Node, UpdateEl, View},
+    li, ol,
+    prelude::IndexMap,
+};
+
+use std::cmp::PartialEq;
+use std::collections::LinkedList;
+
+pub struct Model<Crumb> {
+    crumbs: LinkedList<Crumb>,
```

Comment:
Does it make sense for the Model to be generic at this point? We only have a single contained type, correct? Should we wait for some variant usage before making it generic?

Reply:
I was trying to isolate the logic of breadcrumbs. Not to try to make it generic, but to write minimum code require for it. And I think I have succeeded with lats update. Being generic is a side effect, no more :)

Reply:
Also at the beginning I wasn't sure what a crumb would be. I started with a string, but had in mind some struct. Then I found that the Page enum was also a good choice. It may change in the future, but it does no matter with generic approach.

Reply:
Ok, fair enough :)